src/libraries/System.Runtime/tests/System/Runtime/CompilerServices/RuntimeHelpersTests.cs

```diff
@@ -70,22 +70,6 @@ public static unsafe void GetObjectValue()
             Assert.Equal(i, (int)iOV);
         }
 
-        [Fact]
-        public static unsafe void OffsetToStringData()
-        {
-            // RuntimeHelpers.OffsetToStringData
-            char[] expectedValues = new char[] { 'a', 'b', 'c', 'd', 'e', 'f' };
-            string s = "abcdef";
-
-            fixed (char* values = s) // Compiler will use OffsetToStringData with fixed statements
```

Comment:
The `fixed` keyword doesn't call `OffsetToStringData` any more, so this test wasn't actually testing the right thing. I'm deleting this test since we already have coverage over in __StringTests.cs__:

https://github.com/dotnet/runtime/blob/61733e21bbe965977827205639bf4b835b522494/src/libraries/System.Runtime/tests/System/StringTests.cs#L871-L895